middleware/auth.js

```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+
+export default (req, res, next) => {
+  const { authorization } = req.headers;
+  if (!authorization) {
+    return res.status(401).json({
+      success: false,
+      message: 'You are required to login to access this endpoint',
```

Comment:
I think this error message should be something the user can understand. For example, `Unauthorized! You are required to be logged in to perform this operation.`. We have to assume users know nothing about endpoints.

Reply:
Totally understand, I will implement this immediately.